src/topics/recent.js

```diff
@@ -63,6 +64,10 @@ module.exports = function(Topics) {
 
 	Topics.updateRecent = function(tid, timestamp, callback) {
 		callback = callback || function() {};
-		db.sortedSetAdd('topics:recent', timestamp, tid, callback);
+		if (plugins.hasListeners('filter:topics.updateRecent')) {
+			plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, callback);
```

Comment:
I was thinking, the plugin may not want to override the default sortedSetAdd, and could return the data back.

```
if (plugins.hasListeners('filter:topics.updateRecent')) {
    plugins.fireHook('filter:topics.updateRecent', {tid: tid, timestamp: timestamp}, function(err, data) {
        if (data && data.tid && data.timestamp) db.sortedSetAdd('topics:recent', data.timestamp, data.tid);
        callback(err);
    });
}
```